openpype/hosts/fusion/plugins/load/load_clip.py

```diff
@@ -126,32 +130,31 @@ def loader_shift(loader, frame, relative=True):
     return int(shift)
 
 
-class FusionLoadSequence(load.LoaderPlugin):
-    """Load image sequence into Fusion"""
+class FusionLoadClip(load.LoaderPlugin):
```

Comment:
Actually - I think this will be a major backwards compatible break. I believe the loader gets stored on the container and for that it stores the class's name as far as I know. I'm pretty sure this will break things.

It's imprinted [here](https://github.com/ynput/OpenPype/blob/5c8bbe28713aaabbabf99b4739f1654a0bea4b71/openpype/hosts/fusion/api/pipeline.py#L254) with the [value from e.g. here](https://github.com/ynput/OpenPype/blob/75ea09e238d684b6984e554fd95d0e524f4c036a/openpype/hosts/fusion/plugins/load/load_sequence.py#L167).

This will need to be reverted.

Reply:
Thanks for pointing this out. Will revert back!